### PR TITLE
Fix ESLint issues

### PR DESCRIPTION
**Pull request by nienkedekker**
What it says on the tin 😌

### CODE REVIEW

**Review by karlstolley — APPROVED** [review]
LGTM. The tin and its contents check out.

Happiness is a well-linted set of files. Bonus points for the redundant-looking but essential accessibility move of `<button type="button">`.

File: .eslintrc.js [hunk 1]
```diff
@@ -10,22 +10,14 @@ module.exports = {
     sourceType: 'module',
   },
   plugins: ['react', 'prettier'],
-  // todo: fix these in the code, then remove the rules
```
Comment (jamsea) [inline .eslintrc.js:13]: 🎉

File: src/components/Chat/Chat.js [hunk 2]
```diff
@@ -63,7 +63,7 @@ export default function Chat({ showChat, toggleChat }) {
 
   return showChat ? (
     <aside className="chat">
-      <button onClick={toggleChat} className="close-chat">
+      <button onClick={toggleChat} className="close-chat" type="button">
```
Comment (jamsea) [inline src/components/Chat/Chat.js:66]: I'm surprised we need `type="button"` here 🤔 If it's some sort of a11y rule just having the `button` tag should be enough. Not a big deal either way

File: src/components/HairCheck/HairCheck.js [hunk 3]
```diff
@@ -54,72 +55,68 @@ export default function HairCheck({ joinCall, cancelCall }) {
...
-          <button onClick={join} type="submit">
-            Join call
-          </button>
-          <button onClick={cancelCall} className="cancel-call">
-            Back to start
-          </button>
-        </form>
-      )}
-    </>
+  return getUserMediaError ? (
```
Comment (jamsea) [inline src/components/HairCheck/HairCheck.js:58]: 👍

File: src/components/Chat/Chat.js (hunk 2 above)
Comment (jamsea) [reply src/components/Chat/Chat.js:66]: Nevermind, today I learned: https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/button-has-type.md

**Review by jamsea — APPROVED** [review]

**Pull request merged**